kms/api-client/snippets_test.py

```diff
@@ -177,31 +179,39 @@ def test_key_policy(self):
                                                  self.symId,
                                                  self.member,
                                                  self.role)
-        policy = snippets.get_crypto_key_policy(self.project_id,
-                                                self.location,
-                                                self.keyring_id,
-                                                self.symId)
-        found = False
-        for b in list(policy.bindings):
-            if b.role == self.role and self.member in b.members:
-                found = True
-        assert found
+
+        def check_policy():
+            policy = snippets.get_crypto_key_policy(self.project_id,
+                                                    self.location,
+                                                    self.keyring_id,
+                                                    self.symId)
+            found = False
+            for b in list(policy.bindings):
+                if b.role == self.role and self.member in b.members:
+                    found = True
+            assert found
+        eventually_consistent.call(check_policy,
+                                   exceptions=[Aborted, AssertionError])
         # remove member
         snippets.remove_member_from_crypto_key_policy(self.project_id,
                                                       self.location,
                                                       self.keyring_id,
                                                       self.symId,
                                                       self.member,
                                                       self.role)
-        policy = snippets.get_crypto_key_policy(self.project_id,
-                                                self.location,
-                                                self.keyring_id,
-                                                self.symId)
-        found = False
-        for b in list(policy.bindings):
-            if b.role == self.role and self.member in b.members:
-                found = True
-        assert not found
+
+        def check_policy():
+            policy = snippets.get_crypto_key_policy(self.project_id,
```

Comment:
Same as previous comment, consider using all snake case and avoiding the hanging indent.